tests/results/ResultTest.py

```diff
@@ -37,6 +38,30 @@ def test_message_arguments(self):
         with self.assertRaises(KeyError):
             Result('origin', '{msg}', message_arguments={'message': 'msg'})
 
+    def test_result_actions(self):
+        uut = Result('origin', 'msg')
+        self.assertEqual(uut.actions, tuple())
+
+        action_1 = ResultAction()
+
+        uut.actions = action_1
+        self.assertEqual(uut.actions, (action_1,))
+
+        uut.remove_actions(action_1)
+        self.assertEqual(uut.actions, tuple())
+
+        action_2 = ResultAction()
+        action_3 = ResultAction()
+
+        uut.add_actions(action_1)
+        self.assertEqual(uut.actions, (action_1,))
+
+        uut.add_actions(action_2, action_3)
+        self.assertEqual(uut.actions, (action_1, action_2, action_3))
+
+        uut.remove_actions(action_2, action_3)
+        self.assertEqual(uut.actions, (action_1,))
+
```

Comment:
IMO It feels like that this function has too many blank lines. Maybe group them in a better way?